### PR TITLE
docs: Release Mumble 1.5.634

### PR DESCRIPTION
**Pull request by Hartmnt**

### CODE REVIEW

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+[**GitHub release page**](https://github.com/mumble-voip/mumble/releases/tag/v1.5.634) or from within your Windows client or software package
+management system.
+
+<!--more-->
+
+# Mumble 1.5.634
+
+Almost two years after the previous stable release Mumble 1.4.287, a lot of changes have been made to the codebase. With this update, you will surely
+notice some long-standing bugs fixed, as well as some convenient changes that were long overdue. In the time between the major versions, the project has
+exceeded the 10,000 commit mark and now has over 350 contributors. Thanks to all of you who contributed, translated, submitted issues, and tested
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:23]:
```suggestion
exceeded the 10,000 commit mark and has now over 350 contributors. Thanks to all of you who contributed, translated, submitted issues, and tested
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+pre-releases! Without you, the Mumble project would not be possible at this scale ♥
+
+While the changelog is quite long, the amount of notable client changes is relatively small. For this release, we focus on the few main changes.
+For the interested reader, we of course also include a full changelog at the bottom of this page as well.
+
+If you are wondering, why this first 1.5.x release is not version 1.5.0, but 1.5.634, please see
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:29]:
```suggestion
If you are wondering why this first 1.5.x release is not version 1.5.0, but 1.5.634, please see
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+[this old blog post describing our _new versioning scheme_]({{< relref "/blog/2021-12-07-new-versioning-scheme/index.md" >}}) that
+explains the absence of the mentioned version numbers.
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:31]:
```suggestion
explains the absence of the aforementioned version numbers.
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+For those users who wish to develop positional audio plugins - or try to debug them - we have added a convenient "Positional Audio Viewer" to the
+developer context menu. This will show you the positional information of your client and the associated metadata in real-time.
+
+![Positional Audio Viewer](/blog/media/PositionalAudioViewer.png)
+
+### UI and accessibility overhaul
+
+Mumble 1.5 introduces some quality-of-life UI changes alongside the long-awaited accessibility overhaul. In the software context,
+[accessibility features](https://en.wikipedia.org/wiki/Computer_accessibility) (often shortened to "a11y") are considerations and tools meant to
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:51]:
```suggestion
[accessibility features](https://en.wikipedia.org/wiki/Computer_accessibility) (often shortened as "a11y") are considerations and tools meant to
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+### Positional audio improvements
+
+There have been some changes to the positional audio system, especially to better fine-tune the configuration. The maximum distance thresholds
+have been significantly increased. The option to set a minimum volume of 0 has been brought back. And some clever psychoacoustic effects, such as 
+[interaural delay](https://en.wikipedia.org/wiki/Interaural_time_difference) as well as improved blooming will enhance the positional audio experience
+even more.
+
+For those users who wish to develop positional audio plugins - or try to debug them - we have added a convenient "Positional Audio Viewer" to the
+developer context menu. This will show you the positional information of your client and the associated metadata in real-time.
+
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:45]: Let's mention that the feature was previously provided in the form of a standalone program called MumblePAHelper (Mumble Positional Audio Helper).

There's also a video showcasing the feature in Mumble: https://odysee.com/@Mumble:5/positional-audio-viewer-left4dead:f

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+### UI and accessibility overhaul
+
+Mumble 1.5 introduces some quality-of-life UI changes alongside the long-awaited accessibility overhaul. In the software context,
+[accessibility features](https://en.wikipedia.org/wiki/Computer_accessibility) (often shortened to "a11y") are considerations and tools meant to
+reduce the hassle of running a specific application for users with disabilities. Most often, you will find features specifically crafted or tweaked
+for visually impaired users, but other disabilities such as cognitive or motor impairments should also be considered.
+
+We ensured that the entire application is now keyboard navigable and removed any existing focus traps. We also fixed that some context information,
+such as the muted state and deafened state, of other users are now forwarded to screen readers. Also, all existing settings pages, their contents, and
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:56]:
```suggestion
We ensured that the entire application is now keyboard navigable and removed any existing focus traps. We also implemented forwarding of some context information,
such as the muted/deafened state of other users, to screen readers. Also, all existing settings pages, their contents, and
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+any action that were previously only available as a toolbar button are now usable without a mouse.
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:57]:
```suggestion
any actions that were previously only available as a toolbar button are now usable without a mouse.
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+If you are using a screen reader or other accessibility tooling and have any trouble using Mumble 1.5.634, make sure to let us know via our
+[**GitHub issue tracker**](https://github.com/mumble-voip/issues). We want to make sure the application is usable by all.
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:59]:
```suggestion
[**GitHub issue tracker**](https://github.com/mumble-voip/issues). We want to make sure the application is usable by everyone.
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+
+For users regulary connecting to very large Mumble servers, some neat features were added to make life easier:
+The new message thresholding options will make Mumble limit certain notifications when a configurable user limit is reached on the
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:62]:
```suggestion
The new message thresholding options will make Mumble suppress certain notifications when a configurable user limit is reached on the
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+server. Additionally, users are now able to not only hide certain channels explicitly when activating the channel filter, but it is now also possible to pin
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:63]:
```suggestion
server. Additionally, users are now able to not only hide certain channels explicitly when activating the channel filter, but also pin
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 11]
```diff
@@ -0,0 +1,214 @@
...
+channels in the filtered view even when they are empty.
+
+![Message Threshold](/blog/media/MessageThreshold.png)
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:66]:
```suggestion
![Message threshold](/blog/media/MessageThreshold.png)
```

For consistency.

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 12]
```diff
@@ -0,0 +1,214 @@
...
+
+The new inline local volume adjustment slider removes the complexity of using a dedicated popup menu and therefore reduces the required
+clicks for the same action significantly.
+
+![Local volume adjustment](/blog/media/LocalVolumeAdjustment.png)
+
+A lot of new global shortcut actions were added, including many of the menubar options, allowing to control Mumble with ease even when the
+window is minimized or a fullscreen application is running. Furthermore, notification volumes, audio cue volumes, and text-to-speech volumes can
+now be changed separatly.
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:75]:
```suggestion
window is minimized or a fullscreen application is running. Furthermore, the volume of notifications, audio cues and text-to-speech can now be changed separately.
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md [hunk 13]
```diff
@@ -0,0 +1,214 @@
...
+
+There have also been quite a lot UI fixes, such as the chat log scrolling up when a new image was posted, the "channel name links" not
```
Comment (davidebeatrici) [inline hugo/content/blog/2024-05-19-mumble-1-5-634.md:77]:
```suggestion
There have also been quite a lot of UI fixes, such as the chat log scrolling up when a new image is posted, the "channel name links" not
```

**Review by davidebeatrici — CHANGES_REQUESTED** [review]

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md (hunk 1 above)
Comment (Kissaki) [reply hugo/content/blog/2024-05-19-mumble-1-5-634.md:23]: I prefer *now has* over *has now*

I'm not sure *has now* is even correct grammar - it at least changes the meaning/mood a bit

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md (hunk 4 above)
Comment (Kissaki) [reply hugo/content/blog/2024-05-19-mumble-1-5-634.md:51]: although maybe less obvious to non-EN-speakers, more fluid sounds abbreviated over shortened

```suggestion
[accessibility features](https://en.wikipedia.org/wiki/Computer_accessibility) (often abbreviated as "a11y") are considerations and tools meant to
```

File: hugo/content/blog/2024-05-19-mumble-1-5-634.md (hunk 1 above)
Comment (Hartmnt) [reply hugo/content/blog/2024-05-19-mumble-1-5-634.md:23]: both are correct grammar, I prefer _now has_

Comment (davidebeatrici) [reply hugo/content/blog/2024-05-19-mumble-1-5-634.md:23]: No problem.

**Comment by davidebeatrici** [comment]
The transparency is weird in [LocalVolumeAdjustment.png](https://github.com/mumble-voip/mumble-www/pull/209/files#diff-abcc6b57cba56917fb2e0426b6555a9e092a9a4784c2663f64a980a3bd2bc01d), did you perhaps use a "color to alpha" method?

**Comment by Hartmnt** [comment]
@davidebeatrici I changed it to a screenshot with a saw tooth like effect. Do you think that one fits better?

**Comment by davidebeatrici** [comment]
Yup, I believe it's clear enough that the context menu is cut on purpose.

**Review by davidebeatrici — APPROVED** [review]

**Comment by Hartmnt** [comment]
Updated the other screenshots to match the 1.4 release style (drop shadow and border)

**Pull request merged**